<launch>

  <!-- Load joint controller configurations -->
  <rosparam file="$(find r_mini_driver)/config/controller.yaml" command="load"/>

  <!-- Load joint_position_controllers -->
  <node name="controller_spawner"
	pkg="controller_manager" type="spawner" respawn="false"
	args="joint_state_controller
	      joint1_position_controller
	      joint2_position_controller
	      joint3_position_controller
	      joint4_position_controller
	      joint5_position_controller
        joint6_position_controller"/>

  <!-- Load joint_trajectory_controller -->
  <!--node name="controller_spawner" pkg="controller_manager"
	type="spawner"	respawn="false"
	args="joint_state_controller
	      joint_trajectory_controller"/-->

  <node name="robot_state_publisher"
	pkg="robot_state_publisher" type="robot_state_publisher"/>
  
</launch>
